File: datasets/IR-Plag/original-java-rest-kotlin/case-02/plagiarized/L6/05/inputoutput.kt
package IR

import java.util.*

internal object inputoutput {
    var rad: Double = 0.0
    var panjang: Double = 0.0
    var masukan: Scanner = Scanner(System.`in`)
    fun input() {
        rad = masukan.nextDouble()
        panjang = masukan.nextDouble()
    }

    fun hitArea(): Double {
        return rad.pow(2.0) * 3.14159
    }

    @JvmStatic
    fun main(args: Array<String>) {
        // masukan silinder

        print("Enter the radius and length of a cylinder: ")
        input()

        val vol: Double
        //perhitungan
        val a = hitArea()
        vol = a * panjang


        // cetak hasil
        println("The area is $a")
        println("The volume of the cylinder is $vol")
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L6/05/inputoutput.py
import math

class inputoutput:
    rad = 0.0
    panjang = 0.0

    @staticmethod
    def input():
        inputoutput.rad = float(input("Enter the radius and length of a cylinder: "))
        inputoutput.panjang = float(input())

    @staticmethod
    def hitArea():
        return math.pow(inputoutput.rad, 2) * 3.14159

def main():
    # masukan silinder
    inputoutput.input()

    vol = 0.0
    a = inputoutput.hitArea()

    # perhitungan
    vol = a * inputoutput.panjang

    # cetak hasil
    print("The area is", a)
    print("The volume of the cylinder is", vol)

if __name__ == "__main__":
    main()

